src/map/lua/lua_baseentity.cpp

```diff
@@ -16631,6 +16631,25 @@ void CLuaBaseEntity::useMobAbility(sol::variadic_args va)
     // clang-format on
 }
 
+/************************************************************************
+ *  Function: getAbilityDistance()
+ *  Purpose : Returns the distance for a specified ability from mob_skills
+ *  Example : mob:getAbilityDistance(740)
+ *  Notes   : Uses Ability ID Only
+ ************************************************************************/
+float CLuaBaseEntity::getAbilityDistance(uint16 skillID)
+{
+    auto* PMobSkill{ battleutils::GetMobSkill(skillID) };
```

Comment:
While this is correct, I think we tend to use assignment for this sort of initialisation around the codebase, same inside of drawIn()

Reply:
Updated to use assignment for the initialization:
```cpp
auto* PMobSkill = battleutils::GetMobSkill(skillID);
```